core/trino-main/src/main/java/io/trino/sql/planner/plan/TableWriterNode.java

```diff
@@ -208,12 +208,14 @@ public static class CreateReference
         private final String catalog;
         private final ConnectorTableMetadata tableMetadata;
         private final Optional<TableLayout> layout;
+        private final boolean replace;
 
-        public CreateReference(String catalog, ConnectorTableMetadata tableMetadata, Optional<TableLayout> layout)
+        public CreateReference(String catalog, ConnectorTableMetadata tableMetadata, Optional<TableLayout> layout, boolean replace)
```

Comment:
`replace` seems unused.
I've tried removing it locally and haven't noticed any failures in the tests.

Reply:
It would help during the plan printer right ?